tests/test_image_types.py

```diff
@@ -0,0 +1,337 @@
+import shutil
+
+import wagtail_factories
+
+from django.conf import settings
+from django.test import override_settings
+from test_grapple import BaseGrappleTest, BaseGrappleTestWithIntrospection
+from wagtail import VERSION as WAGTAIL_VERSION
+from wagtail.images import get_image_model
+
+from grapple.types.images import rendition_allowed
+
+
+class ImageTypesTest(BaseGrappleTestWithIntrospection):
+    @classmethod
+    def setUpTestData(cls):
+        cls.image_model = get_image_model()
+        cls.example_image = wagtail_factories.ImageFactory(title="Example Image")
+        cls.example_image.full_clean()
+        cls.example_image.save()
```

Comment:
Do we need to call `full_save()` or `save()` here? I think factories will save the instance/object by default: https://factoryboy.readthedocs.io/en/stable/reference.html#factory.CREATE_STRATEGY

On NCBI, we only call .save() if we've manipulated the object after it's created - and all tests run there.

Reply:
good spot. was a remnant from `<insert reason that I don't recall now>`